resources/test-data.md

```diff
@@ -260,3 +264,5 @@ To simulate an error message, set description in `POST` Create Payment or
 | BANKIDCL    | Payer cancelled BankId signing                 |
 
 [create-card-purchase]: /payments/card/redirect#step-1-create-a-purchase
+[3DS-emulator-no-dropdown]: /assets/img/3DS-emulator-no-dropdown.png
+[3DS-emulator-with-dropdown]: /assets/img/3DS-emulator-with-dropdown.png
```

Comment:
Codacy found an issue: [[definition-case] Do not use uppercase characters in definition labels](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6100690)